Documentation/app-environment.md

```diff
@@ -22,3 +22,5 @@ Since rkt v1.2.0, rkt gives access to systemd-journald's sockets in the /run/sys
 #### /dev/log
 
 Since rkt v1.2.0, if /dev/log does not exist in the image, it will be created as a symlink to /run/systemd/journal/dev-log.
+[networking-dns]: networking/dns.md
```

Comment:
I tested this link again and it worked for me. I'm not sure what is going on here.